docs/source/tutorials/vision/custom_task_tutorial.rst

```diff
@@ -0,0 +1,280 @@
+====================
+Custom Task Tutorial
+====================
+
+Computer vision is consists of different objectives models are trained for. The desired objective reflects
+on the structure of the data and the possible actions on it.
+
+In Deepchecks the first step before running checks is to create an implementation of `VisionData` which
+standardize the structure of the data for a given computer vision task.
+
+Some of the checks are agnostic to the specific task and will work on all of them, and some are tied to the
```

Comment:
This part should be improved - start by saying what current tasks are supported, then give example for checks that can run for other tasks and explain why is that (the last part is what you explained here).